lib/user.rb

```diff
@@ -0,0 +1,41 @@
+class User < Sequel::Model(:userdetails)
+  User.unrestrict_primary_key
+  WORD_LIST = File.readlines(ENV['WORD_LIST_FILE']).map(&:strip)
+
+  def generate(email:)
```

Comment:
Looks more like `find_or_create` than `generate`.

Reply:
`find_or_create` is a method in both ActiveRecord and Sequel, I'd rather avoid any potential conflict/confusion by using it here